traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/util/AbstractResourceWatcher.java

```diff
@@ -57,7 +57,7 @@ public void configure(final JsonNode config) {
 		}
 
 		final WatcherConfig watcherConfig = new WatcherConfig(getWatcherConfigPrefix(), config, trafficOpsUtils);
-		final String resourceUrl = (watcherConfig.getUrl() != null && !watcherConfig.getUrl().isEmpty()) ? watcherConfig.getUrl() : getDataBaseURL();
+		final String resourceUrl = (watcherConfig.getUrl() != null && !watcherConfig.getUrl().isEmpty()) ? watcherConfig.getUrl() : getDefaultDatabaseURL();
```

Comment:
We set `dataBaseURL` later in this function but use the old value of `dataBaseURL between here and there. For those cases, would it make sense to use the condition in this ternary expression to conditionally set `dataBaseURL` to `defaultDatabaseURL` right before this line?

Reply:
i only see it used below in a conditional to check if it has been changed and then it gets set at the bottom to the new value if needed. am i missing other uses?

Reply:
You're right, the old value is fine there.